app/services/data_sync/elasticsearch/user.rb

```diff
@@ -0,0 +1,51 @@
+module DataSync
+  module Elasticsearch
```

Comment:
or just `Search` ? I vaguely remember the idea of using "search" instead of ES around the code but I don't remember the criteria between choosing one or the other

Reply:
The reason I used this here was that I wanted it to be clear what data store we are syncing since I thought that was important. But maybe it isn't. 🤔